src/components/TextArea/TextArea.jsx

```diff
@@ -10,16 +10,23 @@ const TextArea = ({
   error,
   wrapperClassNames,
   inputClassNames,
+  wrapped,
   ...props
 }) => {
-  return (
-    <div className={classNames(styles.textAreaWrapper, wrapperClassNames)}>
+  return !wrapped ? (
+    <div
+      className={classNames(
+        styles.textAreaWrapper,
+        wrapperClassNames,
+        wrapped && styles.fullHeight
```

Comment:
Actually I have a second thought here, I don't think this is needed also, I had something similar with the proposal filed input components, in some places I needed to override nested level of pi-ui's components, and the proper way to achieve that would be on `pigui` use something like:
`wrapperClassNames > div { full height or any styling rule you would like to override }` and passing `wrapperClassNames` should to the trick!

Reply:
see: https://github.com/decred/politeiagui/blob/master/src/components/ProposalForm/ProposalForm.module.css#L27-L29

```
.typeSelectWrapper > div {
  height: 100%;
}
```

Reply:
Good observation!